experimental/skills/DialogToDialog/DialogRootBot/Dialogs/MainDialog.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Integration.AspNet.Core.Skills;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+
+namespace Microsoft.BotBuilderSamples.DialogRootBot.Dialogs
+{
+    /// <summary>
+    /// A sample MainDialog that uses SkillDialog to call skills.
```

Comment:
```suggestion
    /// The main dialog for this bot. It uses skill dialogs to call skills.
```

Reply:
Updated as follows:
/// The main dialog for this bot. It uses a <see cref="SkillDialog"/> to call skills.

---
In reply to: [356213235](https://github.com/microsoft/BotBuilder-Samples/pull/2032#discussion_r356213235) [](ancestors = 356213235)